src/test/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,55 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.commons.util.StringUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    public static final String DEFAULT_DELIIMITER = "[,:]";
+    public static final Pattern CUSTUM_DELIIMITER = Pattern.compile("//(.)\n(.*)");
+
+    @Test
+    @DisplayName("문자열을 자르고 더하는 계산기 기능")
+    public static int splitAndSum(String text) {
```

Comment:
해당 어노테이션들은 테스트 코드에서만 사용해주시면 될 것 같습니다.
```suggestion
    public static int splitAndSum(String text) {
```